decision/ewma.go

```diff
@@ -0,0 +1,5 @@
+package decision
+
+func ewma(old, new, alpha float64) float64 {
+	return new*alpha + (1-alpha)*old
```

Comment:
@Stebalien I know I am late to this PR but if you are willing to shoulder the import, I highly recommend using [gonum/floats](gonum.org/v1/gonum/floats) for the multiplications as it is significantly faster than stdlib.

Reply:
I'd rather keep _this_ case simple as I don't think this is going to be a bottleneck.